reactive-review/apps/reactive-speaker/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+# Dev Services
+quarkus.datasource.devservices.image-name = quay.io/redhattraining/do378-postgres:14.1
+quarkus.kafka.devservices.image-name = quay.io/strimzi/kafka:0.27.1-kafka-3.0.0
```

Comment:
Our Kafka image is not Dev Services friendly (takes ages to be created and be in a healthy state)  so that makes the tests flaky and most of the times they fail because Kafka is not ready. I'm updating the vars file.. good catch!